python/katana/plugin.py

```diff
@@ -71,8 +71,8 @@ def load_module(self, fullname):
                 raise ImportError(
                     f"module {fullname} already exists while trying to insert {aliased_name} at that name."
                 )
-        module.__name__ = fullname
-        module.__package__ = fullname
```

Comment:
What was the original intent of having the module change its name to `fullname`? Why not keep the module name as is?

Reply:
Because I was hoping it would make `katana_enterprise.remote` report that it was `katana.remote` to improve error messages and help.